tiatoolbox/tools/patchextraction.py

```diff
@@ -29,63 +29,62 @@
 
 
 class PatchExtractor(ABC):
-    """
-    Class for extracting and merging patches in standard and whole-slide images.
+    """Class for extracting and merging patches in standard and whole-slide images.
 
     Args:
         input_img(str, pathlib.Path, :class:`numpy.ndarray`): input image for
-          patch extraction.
+            patch extraction.
         patch_size(int or tuple(int)): patch size tuple (width, height).
         input_mask(str, pathlib.Path, :class:`numpy.ndarray`, or :obj:`WSIReader`):
-          input mask that is used for position filtering when extracting patches
-          i.e., patches will only be extracted based on the highlighted regions in
-          the input_mask. input_mask can be either path to the mask, a numpy
-          array, :class:`VirtualWSIReader`, or one of 'otsu' and 'morphological'
-          options. In case of 'otsu' or 'morphological', a tissue mask is generated
-          for the input_image using tiatoolbox :class:`TissueMasker` functionality.
+            input mask that is used for position filtering when extracting patches
+            i.e., patches will only be extracted based on the highlighted regions in
+            the input_mask. input_mask can be either path to the mask, a numpy
+            array, :class:`VirtualWSIReader`, or one of 'otsu' and 'morphological'
+            options. In case of 'otsu' or 'morphological', a tissue mask is generated
+            for the input_image using tiatoolbox :class:`TissueMasker` functionality.
         resolution (int or float or tuple of float): resolution at
-          which to read the image, default = 0. Either a single
-          number or a sequence of two numbers for x and y are
-          valid. This value is in terms of the corresponding
-          units. For example: resolution=0.5 and units="mpp" will
-          read the slide at 0.5 microns per-pixel, and
-          resolution=3, units="level" will read at level at
-          pyramid level / resolution layer 3.
+            which to read the image, default = 0. Either a single
+            number or a sequence of two numbers for x and y are
```

Comment:
Nest the variables in `

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/281/commits/d067b29e6e28646aa6552de10919f37690f32c3e